CONTRIBUTING.md

```diff
@@ -2,13 +2,33 @@
 
 ## Table of Contents
 - [Claiming an Issue](#Claiming-an-Issue)
+- [step 1. create an issue](#step-1.-create-an-issue)
+- [step 2 fork the code](#step-2-fork-the-code)
+- [step 3 Make the necessary changes](#step-3-Make-the-necessary-changes)
+- [step 4 Submit a PR](#step-4-Submit-a-PR)
+- [CONGRADUALTIONS](#CONGRADUALTIONS)
 
-## Claiming an Issue [Back to Top](#Table-of-Contents)
-
+# Claiming an Issue [Back to Top](#Table-of-Contents)
 Here are the steps to claiming an issue:
 
-1. Enter a comment that says something to the effect of "I will work on this" 
-2. wait for a 👍🏾 from one of the organizers.
-3. Fork the code in your own repo.
-4. Make the changes
-5. Submit a PR
+## step 1. create an issue [^Back to Top](#Table-of-Contents)
+
+Start with an issue # Enter a comment that says something to the effect of "I will work on this" and wait for the  one of the organizers to assigned you the issue.
+```
+#8 I'll take this issue off your hands
+```
+This makes it so that more than one person does not work on the same project at the same time (only one pull request will be accepted). 
+
+## step 2 fork the code [^Back to Top](#Table-of-Contents)
+after one of the organizers has  assigned you the issue for the code so that you may freely work on it within your own repository
```

Comment:
```suggestion
after one of the organizers has assigned you the issue for the code so that you may freely work on it within your own repository. 

Fork the code using the "Fork" button in the top-right corner.
```